modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile Registry to access product Devfiles
```

Comment:
Registry/registry
What does product mean here? Does it add any value to the user? Do we need it in the title?